content/plugins/openstack.md

```diff
@@ -1147,6 +1145,65 @@ my_network:
             provider:network_type: vxlan
 {{< /gsHighlight >}}
 
+* Ordering NICs in the Openstack plugin can be done in the 1.4 version of the Openstack plugin by simply stating the relationships to the various networks (or ports) in the desired order, e.g.:
+{{< gsHighlight  yaml  >}}
+node_templates:
+  server:
+    type: cloudify.openstack.nodes.Server
+    relationships:
+      - target: network1
+        type: cloudify.relationships.connected_to
+      - target: network2
+        type: cloudify.relationships.connected_to
+
+  network1:
+    type: cloudify.openstack.nodes.Network
+    properties:
+      resource_id: network1
+
+  network2:
+    type: cloudify.openstack.nodes.Network
+    properties:
+      resource_id: network2
+{{< /gsHighlight >}}
+  in the example above, network1 will be connected to a NIC preceding the one network2 will - however these wont be eth0/eth1, but rather eth1/eth2 - because by default, the management network will be prepended to the networks list (i.e. it'll be assigned to eth0).
+  to avoid this prepending, one should explicitly declare a relationship to the management network, where the network's represented in the blueprint by an existing resource (using the "use_external_resource" property).
+  this will cause the management network adhere the NICs ordering as the rest of them.
+  example:
+{{< gsHighlight  yaml  >}}
+node_templates:
+  server:
+    type: cloudify.openstack.nodes.Server
+    properties:
+      management_network_name: network2
+    relationships:
+      - target: network1
+        type: cloudify.relationships.connected_to
+      - target: network2
+        type: cloudify.relationships.connected_to
+      - target: network3
+        type: cloudify.relationships.connected_to
+
+  network1:
+    type: cloudify.openstack.nodes.Network
+    properties:
+      resource_id: network1
+
+  network2:
+    type: cloudify.openstack.nodes.Network
+    properties:
+      use_external_resource: true
+      resource_id: network2
+
+  network3:
+    type: cloudify.openstack.nodes.Network
+    properties:
+      use_external_resource: true
+      resource_id: network3
+{{< /gsHighlight >}}
+  in this example, "network2" represents the management network, yet it'll be connected to eth1, while "network1" will take eth0, and "network3" (which also happened to already exist) will get connected to eth2.
+  note that the server's property "management_network_name: network2" is not mandatory for this to work - this was just to make the example clear - yet the management network can also be inferred from the provider context (which is what happens when this property isn't explicitly set). were the provider context to have "network2" set as the management network, this example would've worked just the same with this property omitted.
```

Comment:
I would put the entire management_network_name segment in a Node square (or any other sqaure), and omit it from the example...